src/main/java/org/fjala/gugumber/salesforce/ui/pages/event/EventFormAbstract.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * @(#) EventFormAbstract.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of Jala
+ * Foundation, Inc. ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package org.fjala.gugumber.salesforce.ui.pages.event;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Set;
+
+import org.fjala.gugumber.core.StrategySetter;
+import org.fjala.gugumber.salesforce.entities.Event;
+import org.fjala.gugumber.salesforce.ui.pages.BasePage;
+import org.openqa.selenium.WebElement;
+
+/**
+ * EventFormAbstract class
+ *
+ * @author Raúl Choque
+ * @version 0.0.1
+ */
+public abstract class EventFormAbstract extends BasePage {
+
+    /**
+     * Returns a WebElement by create new Event.
+     *
+     * @return a saveEventBtn web element
+     */
+    public abstract WebElement getSaveEventBtn();
+
+    /**
+     * Creates a new Event with the event information.
+     *
+     * @param event     by create a new event.
+     * @param keysEvent by references the values.
+     */
+    public void createEvent(final Event event, final Set<String> keysEvent) {
+        final HashMap<String, StrategySetter> strategyMap = composesStrategyMap(event);
+        keysEvent.forEach(key -> {
+            strategyMap.get(key).executeMethod();
+            System.out.println("Event UI: " + key);
+        });
+        getSaveEventBtn().click();
+    }
+
+    /**
+     * Return an Strategy Map with the "newEvent" parameter.
+     *
+     * @param event is an instance Event.
+     * @return an instance HashMap with keys and methods to run.
+     */
+    private HashMap<String, StrategySetter> composesStrategyMap(final Event event) {
+        final HashMap<String, StrategySetter> strategyMap = new HashMap<>();
+        strategyMap.put("Assigned To", () -> setAssignedToUser(event.getAssignedToUser()));
```

Comment:
this should be constant 
```suggestion
        strategyMap.put("ASSIGNED_TO", () -> setAssignedToUser(event.getAssignedToUser()));
```